jbpm-test-coverage/src/test/java/org/jbpm/test/regression/task/HumanTaskTest.java

```diff
@@ -230,4 +238,31 @@ public void testInputTransformation() {
         assertProcessInstanceCompleted(pi.getId());
     }
 
+    @Test
+    public void testHumanTaskListener() {
+        KieSession ksession = createKSession(HUMAN_TASK_LISTENER);
+        AtomicBoolean triggered = new AtomicBoolean(false);
+        TaskLifeCycleEventListener listener = new DefaultTaskEventListener() {
+            @Override
+            public void afterTaskActivatedEvent(TaskEvent event) {
```

Comment:
beforeTaskActivatedEvent could also be implemented to be tested with a different AtomicBoolean

Reply:
changing this to watch how many times is triggered.